packages/documentation-site/patternfly-docs/content/design-tokens/design-tokens.md

```diff
@@ -0,0 +1,79 @@
+---
+id: About
+section: design-tokens
+---
+
+# Design tokens 
+
+This guide outlines PatternFly's use of design tokens. You should familiarize yourself with the details across this guide to learn about the purpose and implementation of PatternFly's tokens. 
+
+PatternFly's tokens are only available within Figma, so it's important that you follow our [Figma migration](/design-tokens/migrate) guide if necessary.
```

Comment:
This is a little confusing for a coder to read - and a selling point is that tokens cross design/dev. Can you reword this if what you mean is that they aren't in Sketch, so for design you'll need to use Figma?